Jenkinsfile

```diff
@@ -3,59 +3,99 @@ properties(
     parameters(
       [
         string(
-          defaultValue: 'x86_64,ppc64le',
```

Comment:
for now it's fine to remove this from the default, but once my team finishes setting up our multiarch pool, I will likely add more arches to the defaults as we get them